src/Components/quiz/play/Quiz.jsx

```diff
@@ -173,45 +177,46 @@ class Quiz extends Component {
                 break;
             }
         }
-        
+        s
         this.validate(changeArray);
         makeInvestment(ROOMID, this.state.userID, DAY_INDEX, changeArray);
         alert('investment recorded');
     }
- 
-    validate = async (changeArray) => {
-        // TODO: use parent component's access to price to verify onSubmit
-        
-        //const res = await verifyOk(ROOMID, this.state.userID, DAY_INDEX, changeArray, /*put price here*/);
-        //if (!res) {
-        //    alert('You attempted to make an invalid trade. Please reenter.');
-        //}
+
+    //TODO make this implement buyArray instead of just buying one share of the first symbol
```

Comment:
// TODO(xjack[or other ldap here]): make this implement buyArray instead of just buying one share of the first symbol